docs/Community/Downloads.md

```diff
@@ -32,11 +32,13 @@ It is essential that you verify the integrity of release downloads. See [instruc
 From [Maven Central](https://search.maven.org/search?q=g:%20org.apache.datasketches).
 
 ## Version Numbers
-Apache DataSketches uses [semantic versioning](https://semver.org/). Version numbers use the form major.minor.incremental and are incremented as follows:
+Apache DataSketches uses [semantic versioning](https://semver.org/). Version numbers use the form major.minor.incremental and are updated as follows:
 
-* __major version__ for major new functionality and/or major API changes that may be incompatible with prior versions
-* __minor version__ for new functionality and scheduled bug fixes. This should be API compatible with prior versions
-* __incremental version__ for unscheduled bug fixes only
+* __major version__ for major new functionality and/or major API changes that may (or may not) be API compatible with prior versions. 
+* __minor version__ for new functionality and scheduled bug fixes. These should be API compatible with prior versions.
+* __incremental version__ for unscheduled bug fixes only.
+
+Note: we maintain *binary compatiblity* of a specific sketch type across language and time, independent of releases.  For example, a Theta Sketch version 0.1.0 (Aug, 2015) generated using Java and stored to its binary image can be merged with a Theta Sketch version 1.2.0 (Jan, 2020) generated with C++ and stored to its binary image.
```

Comment:
It's specifically forward-compatibilit, no? The newer versions can read the older versions, not the other way around.